### PR TITLE
Align parameter names in ASR attacks

### PR DESCRIPTION
**Pull request by minhitbk**
# Description
This PR is to align parameter names in ASR attacks (Pytorch vs. Numpy versions), it is to make sure that the 2 attacks are interface identical.

## Type of change

Please check all relevant options.

- [x] Improvement (non-breaking)
- [ ] Bug fix (non-breaking)
- [ ] New feature (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Testing

Please describe the tests that you ran to verify your changes. Consider listing any relevant details of your test configuration.

- [ ] Test A
- [ ] Test B

**Test Configuration**:
- OS
- Python version
- ART version or commit number
- TensorFlow / Keras / PyTorch / MXNet version

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by minhitbk** [comment]
This PR was already merged with PR931 which was approved but not merged yet. The purpose is for testing. Therefore, a few commits and code will reoccur.

File: art/attacks/evasion/imperceptible_asr/imperceptible_asr_pytorch.py [hunk 1]
```diff
@@ -85,22 +85,22 @@ class ImperceptibleASRPyTorch(EvasionAttack):
...
-        optimizer_1st_stage: Optional["torch.optim.Optimizer"] = None,
-        optimizer_2nd_stage: Optional["torch.optim.Optimizer"] = None,
-        global_max_length: int = 10000,
+        eps: float = 0.05,
+        max_iter_1: int = 10,
+        max_iter_2: int = 4000,
+        learning_rate_1: float = 0.001,
+        learning_rate_2: float = 5e-4,
+        optimizer_1: "torch.optim.Optimizer" = "torch.optim.Adam",
+        optimizer_2: "torch.optim.Optimizer" = "torch.optim.Adam",
```
Comment (beat-buesser) [inline art/attacks/evasion/imperceptible_asr/imperceptible_asr_pytorch.py:94]: Did you intend `"torch.optim.Adam"` to be a string?

**Review by beat-buesser — CHANGES_REQUESTED** [review]
Hi @minhitbk It looks good to me. Could you please take a look at my question above?

File: art/attacks/evasion/imperceptible_asr/imperceptible_asr_pytorch.py (hunk 1 above)
Comment (minhitbk) [reply art/attacks/evasion/imperceptible_asr/imperceptible_asr_pytorch.py]: It's fixed.

**Review by beat-buesser — APPROVED** [review]

**Pull request merged**